utils/s2n_safety.h

```diff
@@ -93,26 +93,6 @@
  */
 #define ENSURE_NONNULL( x )                          ENSURE((x) != NULL, S2N_ERR_NULL)
```

Comment:
I think we want to replace all of the `NONNULL` macros with these new ones. This change removes the refutation and makes it an assertion, which is in line with the other `ENSURE` macros. Can you remove all of those and update their callers?

Reply:
Done.